src/Apps/LlamaAirforce/Pages/Bribes/Rounds/Rounds.vue

```diff
@@ -113,7 +112,26 @@ const onSelectProtocol = async (
     return;
   }
 
-  store.selectedProtocol = protocol;
+  if (store.selectedPlatform) {
```

Comment:
Bit of a code dupe from `Overview.vue` but it's alright, can worry about that later.